### PR TITLE
Added namespace mode chaos workflow for Hello World application to be triggered from litmus portal.

### PR DESCRIPTION
**Pull request by ishangupta-ds**
Signed-off-by: ishangupta-ds <ishan.gupta@mayadata.io>

*Contains a pod-delete experiment for hello-world app deployment.*

### CODE REVIEW

**Comment by ishangupta-ds** [comment]
@ksatchit @ajeshbaby PTAL.

File: workflows/namespaced-scope-chaos/workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+metadata:
+  generateName: argowf-chaos-hello-world-pod-delete-
+spec:
+  entrypoint: argowf-chaos
+  serviceAccountName: argo-chaos
+  arguments:
+    parameters:
+      - name: adminModeNamespace
+        valueFrom:
+          configMapKeyRef:
```
Comment (ksatchit) [inline workflows/namespaced-scope-chaos/workflow.yaml:12]: Where is the reference for this configmap

Comment (ishangupta-ds) [reply workflows/namespaced-scope-chaos/workflow.yaml:12]: This workflow can be triggered only from litmus-portal and is specifically designed for the namespaced mode of operation. The config map we are creating as a part of the portal installation. @ksatchit

**Review by ksatchit — DISMISSED** [review]

**Comment by ksatchit** [comment]
Lets add a reference link to the configmap spec in the portal as a  comment on the workflow

**Comment by ishangupta-ds** [comment]
@ksatchit PTAL.

**Comment by ksatchit** [comment]
Depends on https://github.com/litmuschaos/litmus/pull/2187. Merging for tests

**Pull request merged**